routing_table.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	log "github.com/golang/glog"
+	"github.com/spikebike/nettools"
```

Comment:
I've changed this to nictuku/nettools. I hope that doesn't break your change.

When doing changes like this, use the original repository in the import line - go get the original repo, then add your fork as a remote so you can work locally with those changes.